Cargo.toml

```diff
@@ -5,10 +5,10 @@ authors = ["Xargin <cao1988228@163.com>"]
 edition = "2018"
 
 [dependencies]
-capstone = "^0.6.0"
-keystone = "0.9.0"
-unicorn = "0.9.1"
+capstone = "^0.9.0"
+keystone = {git = "https://github.com/keystone-engine/keystone/"}
+unicorn-engine = {git = "https://github.com/unicorn-engine/unicorn/", branch = "dev"}
```

Comment:
Maybe specify a specific commit for those git dependencies?

Reply:
I want to keep them up to date until they don't work

Reply:
maybe i should specify a specific commit id